docs/plugin_development.rst

```diff
@@ -352,48 +427,66 @@ After ``cookicutter`` finishes its job, you'll get the following tree structure
-  On this file the ``entry_points`` parameter of ``setup`` is quite important, as it is the one that allows Spyder to identify this package as a plugin, and to know how to access its functionalities.
+     clean                remove all build, test, coverage and Python artifacts
+     clean-build          remove build artifacts
+     clean-pyc            remove Python file artifacts
+     clean-test           remove test and coverage artifacts
+     test                 run tests quickly with the default Python
+     docs                 generate Sphinx HTML documentation, including API docs
+     servedocs            compile the docs watching for changes
+     release              package and upload a release
+     dist                 builds source and wheel package
+     install              install the package to the active Python's site-packages
+     develop              install the package to the active Python's site-packages
 
+* ``setup.py``.
+  It helps you install, package and distribute your plugin with Setuptools, the standard for distributing Python packages.
+  In this file the ``entry_points`` parameter of ``setup`` is quite important, as it is what allows Spyder to identify this package as a plugin, and know how to access its functionality.
 
-
-The ``spyder-pomodoro-timer`` folder has the name you introduced when running ``cookiecutter``. Inside this you'll see a folder called ``spyder``, where we will place the code of our plugin.
+The folder for ``spyder-pomodoro-timer`` has the project name you entered when you ran ``cookiecutter``.
+Inside it we should see a folder called ``spyder``, where we will place the code of our plugin.
 
 In the ``spyder`` directory you'll find the following files:
 
-* ``api.py``: where the functionality of the plugin is exposed to the rest of Spyder. That would allow additional functionality to be added from other plugins.
+* ``api.py``.
+  Exposes the functionality of the plugin to the rest of Spyder, allowing other plugins to build on top of it in turn.
 
-* ``plugin.py``: is the core of the plugin. Depending on the type of plugin we created, here you'll see an instance of ``SpyderDockablePlugin`` or ``SpyderPluginV2``.
+* ``plugin.py``.
+  This is the core of the plugin.
+  Depending on the type of plugin created, you will see an instance of either ``SpyderDockablePlugin`` or ``SpyderPluginV2``.
 
   * If it is a ``SpyderPluginV2`` you should set a constant class named ``CONTAINER_CLASS`` with an instance of ``PluginMainContainer``.
   * If it is a ``SpyderDockablePlugin`` you should set a constant class named ``WIDGET_CLASS`` with an instance of ``PluginMainWidget``.
+  
+  For example:
+  
+  .. code-block:: python
+     
+     # plugin.py
+     from spyder.api.plugins import SpyderPluginV2
+     from my_plugin.spyder.container import MyContainer
+
+     class MyPlugin(SpyderPluginV2):
+         CONTAINER_CLASS = MyContainer
```

Comment:
```suggestion
  * If it is a ``SpyderDockablePlugin`` you should set a constant class named ``WIDGET_CLASS`` with an instance of ``PluginMainWidget``.

  For example:

  .. code-block:: python

     # plugin.py
     from spyder.api.plugins import SpyderPluginV2
     from my_plugin.spyder.container import MyContainer

     class MyPlugin(SpyderPluginV2):
         CONTAINER_CLASS = MyContainer
```

Fix spurious trailing spaces—made sure this didn't happen elsewhere, just missed a spot here.